Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -364,23 +343,15 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         // from rad to deltaCotTheta
         float p2scatterSigma = pT2scatterSigma * iSinTheta2;
         // if deltaTheta larger than allowed scattering for calculated pT, skip
-        if (m_config.enableCutsForSortedSP) {
-          if (deltaCotTheta2 - error2 > p2scatterSigma) {
-            if (m_config.skipPreviousTopSP) {
-              if (cotThetaB - lt.cotTheta < 0) {
-                break;
-              }
-              t0 = t;
+        if (deltaCotTheta2 - error2 > p2scatterSigma) {
```

Comment:
We need to justify the calculation in a comment:

```
// Apply a cut on the compatibility between the r-z slope of the two seed segments. 
// This is done by comparing the squared difference between slopes, and comparing 
// to the squared uncertainty in this difference - we keep a seed if the difference 
// is compatible within the assumed uncertainties.
// The uncertainties get contribution from the  space-point-related squared error (error2)
// and a scattering term calculated assuming the minimum pt we expect to reconstruct (p2scatterSigma).   
// This assumes gaussian error propagation which allows just adding the two errors 
// if they are uncorrelated (which is fair for scattering and measurement uncertainties)
```

I would suggest the following code change:

```suggestion
        if (deltaCotTheta2 > (error2 + p2scatterSigma) ) {
```

Reply:
done!